### ISSUE TITLE
Use Retrofit

### ISSUE THREAD

**Issue by Fshamri** [phubbard/Meltdown#21]
Hi,
it is good application ... what about using retrofit?

**Comment by phubbard** [comment]
Wow, that’s a nice library. Yeah, that’s a good idea, would simplify the communications code and make it easier to read and maintain too.

Are you interested? Or should I add it to the feature list for when I have time?

**Comment by Fshamri** [comment]
Yeah ... you can use retrofit long with Okhttp and picasso to simplify the communications code also for execution speed.
